Source/WebCore/dom/Node.cpp

```diff
@@ -2499,6 +2500,9 @@ void Node::defaultEventHandler(Event& event)
             }
         }
 #endif
+    } else if (eventType == eventNames.pointerdownEvent || eventType == eventNames.pointerupEvent) {
+        if (is<PointerEvent>(event))
+            document().handlePopoverLightDismiss(event);
```

Comment:
If we are going to do a type check here, then should `handlePopoverLightDismiss` take a `PointerEvent&`?